### ISSUE TITLE
v3.4.4 Swiss Chard

### ISSUE THREAD

**Issue by andrewpbrett** [openfoodfoundation/openfoodnetwork#6448]
Steps: 

- [x] Include translations: `tx pull --force`
- [x] [Draft new release] https://github.com/openfoodfoundation/openfoodnetwork/releases/tag/untagged-f09be9b5dcfd50e284da
- [x] Notify #instance-managers of user-facing changes.
- [x] Test: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master/builds/1938
- [x] Update translations if necessary
- [x] Publish and notify #global-community
- [x] Deploy and notify #instance-managers
- [x] Nudge next release manager

The full process is described at https://github.com/openfoodfoundation/openfoodnetwork/wiki/Releasing.

[Draft new release]: https://github.com/openfoodfoundation/openfoodnetwork/releases/new?tag=v&title=v+Code+Name&body=Congrats%0A%0ADescription%0A%0A%23%23+User+facing+changes+:eyes:%0A%0A%0A%0A%23%23+Technical+changes+:wrench:%0A%0A

**Comment by filipefurtad0** [comment]
Hey @andrewpbrett ,

The release looks good to go:
https://docs.google.com/document/d/1ckuuXjUNdm6Mc37VwUF3c-4Y_VBi9dauCvdZnAtuW5I/edit#

Only issue found is #6459 which is a regression, quite misleading one - may be an S3...  Leaving this PR here for now - we'll check next week how to tackle this.

Other than that - thumbs up!  :+1:

**Comment by filipefurtad0** [comment]
Ok! I went through the Taxes feature, I'd say this is looking very good. Tax-related test notes here (this document is WIP and related to the ongoing adjustments-task):
https://docs.google.com/document/d/1L_x35_si6aqjORLseLdBOaHht6D3Q8ny8h1NOiseq2g/edit#

Found #6462 and #6463 and, as mentioned above, #6459 - none of them seems to be a release blocker. I'm moving the release to Ready to Go - but I guess we'll have the chance to discuss these before deployment - for this reason, I'm adding the feedback label.
